src/lexer.coffee

```diff
@@ -573,7 +573,11 @@ exports.Lexer = class Lexer
 
     [firstToken, ..., lastToken] = tokens
     firstToken[2].first_column -= delimiter.length
-    lastToken[2].last_column += delimiter.length
+    if lastToken[1].substr(-1) == '\n'
```

Comment:
Please use `is` instead of `==`. In CoffeeScript `==` compiles to `===`, so we try to avoid using it in code to avoid confusion. (I know, there are a few places we should fix that use it already, but we shouldn’t add more.)